ambari-logsearch/pom.xml

```diff
@@ -61,6 +61,28 @@
   </repositories>
   <build>
     <plugins>
+       <plugin>
+         <groupId>org.vafer</groupId>
+         <artifactId>jdeb</artifactId>
+         <version>1.0.1</version>
```

Comment:
Having version at a single place like ambari-project/pom.xml is better than putting them at different levels

Reply:
I do agree with you.  That makes more sense.  I have added it different levels because I was facing errors in each of these levels when I issued a command to generate debian package of Ambari from top level.